website/project/model.py

```diff
@@ -3815,6 +3841,8 @@ class DraftRegistration(StoredObject):
 
     approval = fields.ForeignField('draftregistrationapproval', default=None)
 
+    status_logs = fields.ForeignField('draftregistrationlog', list=True)
```

Comment:
Consistent, yes, however, what is the general rationale for this architecture?
- Does having a list of logs imply a new set of automatic queries? If so, why do we even have this option? In what case would this ever be optimal?
- Why is a backref, assuming the previous question is reasonable, any better? Won't it load the DraftRegistration with each log loaded?
-